src/snowflake/connector/aio/_network.py

```diff
@@ -827,6 +827,11 @@ async def _request_exec(
             RuntimeError,
             AttributeError,  # json decoding error
         ) as err:
+            if "Event loop is closed" in str(err):
```

Comment:
can we also check err type here to be more precise?
```suggestion
            if "Event loop is closed" in str(err) and instance(err, ...):
```